### ISSUE TITLE
Function short for adfly.py seems to have an issue.

### ISSUE THREAD

**Issue by greespinal** [ellisonleao/pyshorteners#38]
print "My short url is {}".format(shortener.short(url, uid=12107955, api_key='a87a1a49af0783388e67c3da80eb9f17', type='int'))
TypeError: short() got an unexpected keyword argument 'uid'

**Comment by greespinal** [comment]
Put arguments in shortener variable instead of short() function
shortener = Shortener('AdflyShortener', uid='12107955', key='a87a1a49af0783388e67c3da80eb9f17', type='int')

Result:
My short url is http://adf.ly/1TspsP

**Comment by pouya-barzegar** [comment]
it still does'nt work for me.
i get an error for this code:
>>> from pyshorteners import Shortener
>>> shortener = Shortener('Adfly', uid=uid, api_key=api_key, type='int')
>>> shortener.short("http://google.com")
....
    raise TypeError('Please input the key and uid value')
TypeError: Please input the key and uid value

and when pass api_key and uid to the short() method, it says it doesn't support that. :disappointed:

**Comment by 0xicl33n** [comment]
I  figured it out after reading the code in adfly.py in this repo. I have made it work @pouya-barzegar 

Here 

```
uid = YOUR USER ID #do not use quotes on this, it has to be an integar
key = 'YOUR PUBLIC KEY'
advert_type = 'int'
url = 'URL'
shortener = Shortener('Adfly',key=key,uid=uid,type=advert_type)
shortener.short(url)
```

**Comment by ellisonleao** [comment]
Fixed on 1.0. Make sure to read the docs when it comes available.

**Comment by mauro199304** [comment]
is it possible to update the short url?  what parameters to use?
